src/lib/journey/callbacks/redirect/redirect.svelte

```diff
@@ -0,0 +1,26 @@
+<script lang="ts">
+  import { FRAuth, FRStep, type RedirectCallback } from '@forgerock/javascript-sdk';
+
+  import type {
+    CallbackMetadata,
+    SelfSubmitFunction,
+    StepMetadata,
+  } from '$journey/journey.interfaces';
+  import Spinner from '$components/primitives/spinner/spinner.svelte';
+  import type { Style } from '$lib/style.store';
+  import type { Maybe } from '$lib/interfaces';
+
+  export let callback: RedirectCallback;
+  export let callbackMetadata: CallbackMetadata;
+  export let selfSubmitFunction: Maybe<SelfSubmitFunction> = null;
+  export let stepMetadata: StepMetadata;
+  export let style: Style = {};
+
+  export let step: FRStep;
+
+  FRAuth.redirect(step);
+</script>
+
+<div class="tw_text-center">
+  <Spinner colorClass="tw_text-primary-light" layoutClasses="tw_h-24 tw_mb-6 tw_w-24" />
```

Comment:
I made the decision for this callback to just show a spinner, and not take any text to display. I'm happy to be overruled but I'm not sure what we would show here.

Reply:
We will need to have some text to ensure this is accessible for users of screen readers. We also want to make sure it's read to the user assertively, so we are sure it's been communicated. The ARIA Live attribute with `"assertive"` should do it: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions.

Reply:
I think text of "Redirecting to <top domain>"? Or, something like that.

Reply:
For example: If the user clicks on Google as their Identity Provider, the UI would say the following:

```
Redirecting you to google.com.
```

You could extract the domain from the URL we get from the callback, so it would work quite universally. Just a thought.

Reply:
This makes sense - the domain text is easy enough, and I'll look into the ARIA stuff you mentioned. Thanks.